task_5_2.py

```diff
@@ -0,0 +1,27 @@
+
+
+def is_valid_input(n):
+    if n % 1 != 0:
+        return False
+    return True
```

Comment:
consider the example
```suggestion
def is_valid_input(n):
    return not n % 1 != 0
```